app/views/admin/spaces/index.html.erb

```diff
@@ -48,5 +48,6 @@
       <%= form.button "Save", type: "submit", class: "c-btn c-btn--primary" %>
     </div>
   <% end %>
-
+  <hr />
+  <p>Learn <a href="https://admin.forem.com/docs/advanced-customization/content-manager/spaces">more about Spaces in the Admin Guide</a>.</p>
```

Comment:
```suggestion
  <p>Learn <a href="https://admin.forem.com/docs/advanced-customization/content-manager/spaces" target="_blank" rel="noopener">more about Spaces in the Admin Guide</a>.</p>
```

Just a suggestion, but we could make this open in a new tab, so the user doesn't leave their current context, which could be helpful.

> I chose to go with the following, as assistive technology will announce
the link's text as "more about Spaces in the Admin Guide", which I find
more a11y oriented.

❤️  💯 

I'm wondering if it might visually look more appealing if we made the _entire_ sentence a hyperlink (adding the "Learn" inside the `<a>`) - consider this just a suggestion though, and from the a11y POV either way provides good context for screen reader users, and I love that this was considered 👍